### ISSUE TITLE
Task name isn't displayed in tab during creation neither in breadcumbs menu

### ISSUE THREAD

**Issue by petrenkonikita112263** [xwikisas/application-task#37]
It was tested on XWiki 14.10.6 with the latest version of app 3.0.2, latest MS Edge browser.

**STEPS TO REPRODUCE**

1. Install latest version of the application v3.0.2
2. Get trial license
3. Go to home page of Task Manager
4. Press "Create" and name new task "Task A"
5. Select Task Template
6. Press "Next"

**ACTUAL RESULT**
During the filling fields, in the web-browser tab the name of the task "Task A" isn't visible, it's only exist in URL. It appears only after pressing "Save". Neither in menu.

![image](https://github.com/xwikisas/application-task/assets/56109799/7b0aee54-57ba-47b2-b7fe-7c40568797b5)
![image](https://github.com/xwikisas/application-task/assets/56109799/4e20fc4c-1ae9-4bbe-8f6d-913704183393)
![image](https://github.com/xwikisas/application-task/assets/56109799/a669f260-0e59-4e65-b37c-5d9eb7222ce7)

**EXPECTED RESULT**
The name of the Task is present in web browser tab during the creation process. Same true for breadcrumb menu.
